tests/integration.rs

```diff
@@ -2728,6 +2728,106 @@ mod tests {
             handle_child_output(r, &output);
         }
 
+        #[test]
+        #[cfg(not(feature = "mshv"))]
+        fn test_pci_multiple_segments() {
+            // Prepare another disk file for the virtio-disk device
+            assert!(exec_host_command_status("mkdir -p /tmp/test_pci_multiple_segments").success());
+            assert!(exec_host_command_status(
+                "truncate /tmp/test_pci_multiple_segments/test-disk.raw -s 4M"
+            )
+            .success());
+            assert!(exec_host_command_status(
+                "mkfs.ext4 /tmp/test_pci_multiple_segments/test-disk.raw"
+            )
+            .success());
```

Comment:
I would prefer you wrote into the test's temp directory (`guest.tmp_dir`) and then it will be cleaned up automatically.

Reply:
Sure, will change. Thanks for the suggestion!

Reply:
Done